third_party/misc/brython_time.py

```diff
@@ -0,0 +1,406 @@
+# from browser import window
```

Comment:
I'm a little worried about using this implementation since it depends on Brython-specific functionality. Seems like more effort to work around potential incompatibilities here than to just write something from scratch. I might be underestimating the effort involved though.